src/sketch.py

```diff
@@ -0,0 +1,8 @@
+class FeatureSketch:
+    ...
+
+def Sketch(self, sheetSize: float = 200, gridSize: float = 5) -> FeatureSketch:
+    ...
+
+def SketchTransform():
+    ...
```

Comment:
May I ask why add this, I didn't see any FeatureSketch class and SketchTransform function in the Abaqus documentation

Reply:
I just added these to fill the `sketch` module. They came from inspection in the Abaqus CLI.
Do you think it is better to keep this module as an empty file?

Reply:
It is great to add this, I leave it blank just because I don't know what to add. Inspection from abaqus command line seems a good way to fill these individual modules.